docs/en/latest/tutorials/monitor-api-health-check.md

```diff
@@ -0,0 +1,180 @@
+
+> The simplest and standardized way to validate the status of a service is to define a new [health check](https://docs.microsoft.com/en-us/dotnet/architecture/microservices/implement-resilient-applications/monitor-app-health) endpoint like `/health` or `/status`
+
+## Setting Up Health Checks in APISIX
+
+This process involves checking the operational status of the 'upstream' nodes. APISIX provides two types of health checks: **Active checks** and **Passive Checks** respectively. Read more about Health Checks and how to enable them [here](https://apisix.apache.org/docs/apisix/tutorials/health-check/). Use the [Admin API](https://apisix.apache.org/docs/apisix/admin-api/) to create an Upstream object. Here is an example of creating an [Upstream](https://apisix.apache.org/docs/apisix/terminology/upstream/) object with two nodes (Per each backend service we defined) and configuring the health check parameters in the upstream object:
+
+```bash
+curl "http://127.0.0.1:9180/apisix/admin/upstreams/1" -H "X-API-KEY: edd1c9f034335f136f87ad84b625c8f1" -X PUT -d '
+{
+   "nodes":{
+      "web1:80":1,
+      "web2:80":1
+   },
+   "checks":{
+      "active":{
+         "timeout":5,
+         "type":"http",
+         "http_path":"/health",
+         "healthy":{
+            "interval":2,
+            "successes":1
+         },
+         "unhealthy":{
+            "interval":1,
+            "http_failures":2
+         }
+      }
+   }
+}'
+```
+
+This example configures an active health check on the **`/health`** endpoint of the node. It considers the node healthy after **one successful health check** and unhealthy **after two failed health checks**.
+
+> Note that sometimes you might need the IP addresses of upstream nodes, not their domains (`web1` and `web2`) if you are running services outside docker network. It is by design that the health check will be started only if the number of nodes (resolved IPs) is bigger than 1.
+
+## Enable the Prometheus Plugin for a route
+
+Create a [Route](https://apisix.apache.org/docs/apisix/terminology/route/) object and enable the Prometheus plugin for the route by adding `"prometheus": {}` in the plugins option. APISIX gathers internal runtime metrics and exposes them through port `9091` and URI path `/apisix/prometheus/metrics` by default that Prometheus can scrape. It is also possible to customize the export port and **URI path**, **add** **extra labels, the frequency of these scrapes, and other parameters** by configuring them in the Prometheus configuration `/prometheus_conf/prometheus.yml`file.
```

Comment:
For Prometheus, I believe a global rule is better suited

Reply:
Added a global rule too